src/ontology/cl-edit.owl

```diff
@@ -21153,10 +21153,10 @@ SubClassOf(obo:CL_0004137 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0008925
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:12209831") obo:IAO_0000115 obo:CL_0004138 "A retinal ganglion A cell with dense arbor near soma.")
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004138 "https://orcid.org/0000-0001-7258-9596")
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004138 "https://orcid.org/0000-0002-5260-9315")
-AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_0004138 "type I")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0004138 "I-1")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0004138 "outer alpha retinal ganglion cell")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0004138 "cell")
+AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:CL_0004138 "I-1")
```

Comment:
Consider adding abbreviation tag to "I-1".

Reply:
together with retinal ganglion cell A1, I will investigate them elsewhere. Both have as synonym Type I, and the related synonyms don't make sense.